vm-setup/roles/v1aX_integration_test/vars/main.yml

```diff
@@ -65,6 +65,32 @@ IMAGE_NAME_CENTOS: "{{ lookup('env', 'IMAGE_NAME') | default('CENTOS_8.2_NODE_IM
 RAW_IMAGE_NAME_CENTOS: "{{ lookup('env', 'IMAGE_RAW_NAME') | default('CENTOS_8.2_NODE_IMAGE_K8S_v1.18.8-raw.img', true)}}"
 IMAGE_LOCATION_CENTOS: 'https://artifactory.nordix.org/artifactory/airship/images/k8s_v1.18.8/'
 
+# Environment variables to install ironic
+GOPATH: "{{ lookup('env', 'GOPATH')}}"
+BMOPATH: "{{ lookup('env', 'BMOPATH') | default('{{ GOPATH }}/src/github.com/metal3-io/baremetal-operator', true)}}"
+IRONIC_TLS_SETUP: "{{ lookup('env', 'IRONIC_TLS_SETUP') | default('true', true)}}"
+IRONIC_BASIC_AUTH: "{{ lookup('env', 'IRONIC_BASIC_AUTH') | default('true', true)}}"
+DEPLOY_BMO: "{{ lookup('env', 'DEPLOY_BMO') | default('false', true)}}"
+DEPLOY_IRONIC: "{{ lookup('env', 'DEPLOY_IRONIC') | default('true', true)}}"
+DEPLOY_TLS: "{{ lookup('env', 'IRONIC_TLS_SETUP') | default('true', true)}}"
+DEPLOY_BASIC_AUTH: "{{ lookup('env', 'IRONIC_BASIC_AUTH') | default('true', true)}}"
+DEPLOY_KEEPALIVED: "{{ lookup('env', 'DEPLOY_KEEPALIVED') | default('true', true)}}"
+
+# TLS environment variables
+IRONIC_CA_CERT_B64: "{{ lookup('env', 'IRONIC_CA_CERT_B64')}}"
+IRONIC_CACERT_FILE: "{{ lookup('env', 'IRONIC_CACERT_FILE')}}"
+IRONIC_CERT_FILE: "{{ lookup('env', 'IRONIC_CERT_FILE')}}"
+IRONIC_KEY_FILE: "{{ lookup('env', 'IRONIC_KEY_FILE')}}"
+IRONIC_INSPECTOR_CACERT_FILE: "{{ lookup('env', 'IRONIC_INSPECTOR_CACERT_FILE')}}"
+IRONIC_INSPECTOR_CERT_FILE: "{{ lookup('env', 'IRONIC_INSPECTOR_CERT_FILE')}}"
+IRONIC_INSPECTOR_KEY_FILE: "{{ lookup('env', 'IRONIC_INSPECTOR_KEY_FILE')}}"
+
+# Basic Auth environment variables
+IRONIC_USERNAME: "{{ lookup('env', 'IRONIC_USERNAME')}}"
+IRONIC_PASSWORD: "{{ lookup('env', 'IRONIC_PASSWORD')}}"
+IRONIC_INSPECTOR_USERNAME: "{{ lookup('env', 'IRONIC_INSPECTOR_USERNAME')}}"
+IRONIC_INSPECTOR_PASSWORD: "{{ lookup('env', 'IRONIC_INSPECTOR_PASSWORD')}}"
```

Comment:
those would be set only if tls_setup.sh and basic-auth.sh have been sourced before calling ansible, i.e. in the script that calls ansible

Reply:
Anyways those scripts will be sourced in 03_* and will be set.

Reply:
no, because the ansible scripts for provisioning are independent of 03_* they can be called on their own, or as part of 05_*

Reply:
I am sourcing tls_setup.sh and basic-auth.sh before calling ansible now, do I need to still source them in 05_* ?